src/preFilter.ts

```diff
@@ -18,14 +19,18 @@ export function filterByContent(searchee: Searchee): boolean {
 		});
 	}
 
-	const isSingleEpisodeTorrent =
-		searchee.files.length === 1 && EP_REGEX.test(searchee.files[0].name);
-
-	if (!includeEpisodes && isSingleEpisodeTorrent) {
+	const isSingleEpisodeTorrent = searchee.files.length === 1 && EP_REGEX.test(searchee.name); 
+	const isSeasonPack = typeof searchee.path !== 'undefined' ? SEASON_REGEX.test(path.dirname(searchee.path).split(path.sep).reverse()[0]): false;
+	
+	if (!includeEpisodes && isSingleEpisodeTorrent && !isSeasonPack) {
```

Comment:
oh, I see what you're saying now about `fallback(fileConfig.includeEpisodes)` now.

I think the best way to make it backwards compatible will probably be to invert the option - instead of a switch to include season pack episodes (when really we want it to be exclude), maybe we can have an option to include standalone episodes (not from a season pack). 

that way, we can preserve the original behavior of `includeEpisodes` but still satisfy the use case. 

I'm also not totally opposed to just making the change, but curious to hear your thoughts

Reply:
I think the "include" theme for the content filtering is something to potentially maintain, as we don't have any other excluding (content-based) config options. I'm not really opposed to changing to exclusion, but for mere consistency, and maybe aesthetic of the content config portion, it would seem logical.

Current behavior with `includeEpisodes = true` would essentially be the same with falling-back to fileConfig.includeEpisodes. 

I see this as the optimal way to ensure consistency of behavior for those who don't actively watch changelog/commits in version updates, and would otherwise have the introduction of includeSeasonPackEpisodes go completely unnoticed. 

I also assume most people would not be fussing with their configuration file regularly, even fewer are probably looking out for potentially adding new options. 

Set it and forget it situation. Update the container when notified, but never read changelog.

`fallback(fileConfig.includeSeasonPackEpisodes, fileConfig.includeEpisodes)`

Reply:
i'm marking this resolved due to the change to includeSingleEpisodes